vendordeps/REVLib.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
@rice31415 Do you think that adding the vendor libraries may cause a git conflict since the other branches probably have this committed change as well?

Reply:
Nothing else has used a Spark MAX yet so I don't think the other branches actually have the REV dependency yet, but the climber might also have this change once that gets worked on